### PR TITLE
Add blog posts to homepage

### PR DESCRIPTION
**Pull request by BrandonKlotz**
Adds Recent News section to the home page, limiting to 4 posts.

Screen Shot 2020-04-18 at 12 09 19 PM

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -110,6 +110,7 @@ params:
       job: Commissioner, City of Grand Rapids
       img: 4
   section5: true
+  section6: true
```
Comment (BrandonKlotz) [inline config.yaml:113]: This tells the `hugo-fresh` theme to look for `layouts/section6.html`

File: layouts/_default/list.html [hunk 2]
```diff
@@ -5,35 +5,28 @@
...
-    {{ range .Pages }}
-        <div class="tile">
-            <a href="{{.Permalink}}">{{.Date.Format "2006-01-02"}} | {{.Title}}</a>
-        </div>
-    {{ end }}
-</div>
-
-<br>
+    {{ $posts := (where .Site.RegularPages "Section" "blog") }}
+    {{ partial "news.html" $posts }}
```
Comment (BrandonKlotz) [inline layouts/_default/list.html:16]: I pulled out the `range` of posts from this partial into a `news.html` partial so we can feed it all blog posts on the News page and limit what we feed the partial on the Homepage.

File: layouts/partials/news.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="container rows">
+    {{ range . }}
```
Comment (BrandonKlotz) [inline layouts/partials/news.html:2]: This `.` is what we are passing into the partial. In the example below it is the blog posts data limited by 4.

```
{{ $posts := (where .Site.RegularPages "Section" "blog" | first 4) }}
{{ partial "news.html" $posts }}
```

File: layouts/_default/list.html (hunk 2 above)
Comment (Allen616) [reply layouts/_default/list.html:16]: Cool, that is what I wanted.

File: layouts/partials/news.html (hunk 3 above)
Comment (Allen616) [reply layouts/partials/news.html:2]: ahh.

File: config.yaml (hunk 1 above)
Comment (Allen616) [reply config.yaml:113]: got it.

**Comment by Allen616** [comment]
Will it pull through a selected image to the home page? I am thinking of screen shots of the websites, or something related to the project.

**Comment by Allen616** [comment]
Just a small image, the same size as the project icons on the homepage.

**Review by Allen616 — APPROVED** [review]
Thanks! Looks great. I really appreciate it.

**Pull request merged**